e2e/testdata/fn-eval/image-pull-policy-if-not-present/.expected/setup.sh

```diff
@@ -18,5 +18,5 @@ set -eo pipefail
 # Function gcr.io/kpt-fn-demo/foo:v0.1 prints "foo" to stderr and
 # function gcr.io/kpt-fn-demo/bar:v0.1 prints "bar" to stderr.
 # We intentionally tag a wrong image as pull gcr.io/kpt-fn-demo/bar:v0.1
-docker pull gcr.io/kpt-fn-demo/foo:v0.1
-docker tag gcr.io/kpt-fn-demo/foo:v0.1 gcr.io/kpt-fn-demo/bar:v0.1
+${KPT_FN_RUNTIME} pull gcr.io/kpt-fn-demo/foo:v0.1
```

Comment:
ideally we should have a test where this is not specified and default runtime is used.